web/client/epics/measurement.js

```diff
@@ -86,8 +88,16 @@ export const setMeasureStateFromAnnotationEpic = (action$, store) =>
 export const addCoordinatesEpic = (action$, {getState = () => {}}) =>
     action$.ofType(CLICK_ON_MAP)
         .filter(() => {
-            const {showCoordinateEditor, enabled} = getState().controls.measure;
```

Comment:
@tonym3nsa I think this fix will work only for a MapStore project that use the "desktop"/"mobile" plugin struture, in addition I'm not sure if the embed.

Probably this could be fixed by checking if the `controls.measure` state exist and if not return undefined for `showCoordinateEditor` and `enabled` in this way the filter will be false and the error should disappear.

Could you please verify if this approach could fix the issue instead?
```js
const { showCoordinateEditor, enabled } = getState()?.controls?.measure || {};
return showCoordinateEditor && enabled;
```